### DISCUSSION TITLE
shen-ping/random

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#5] [Announcements]
# shen-ping/random

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/shen-ping/random

**Comment by wurencaideli** [comment]
搞笑神评论，可以投稿哟

**Reply by asdqcmj33** [reply]
建议每个小帖子下有评论功能

**Reply by wurencaideli** [reply]
是有的，点击上面的原文就可以去评论了哟

**Comment by asdqcmj33** [comment]
666

**Comment by asdqcmj33** [comment]
okok，多谢指点！



---Original---
From: ***@***.***&gt;
Date: Fri, Jun 30, 2023 11:39 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [wurencaideli/dumogu-ypwz-comment] shen-ping/random (Discussion#5)




 
是有的，点击上面的原文就可以去评论了哟
